automatic/PDFCreator/update.ps1

```diff
@@ -0,0 +1,24 @@
+import-module au
+
+$url = 'http://download.pdfforge.org/download/pdfcreator/PDFCreator-stable?download'
+
+function global:au_SearchReplace {
+    @{
+        'tools\ChocolateyInstall.ps1' = @{
+            "(^[$]checksum\s*=\s*)('.*')"   = "`$1'$($Latest.Checksum32)'"
+        }
+     }
+}
+
+function global:au_GetLatest {
+    Write-Host $url
+
+    $temp_file = $env:TEMP + '\PDFCreator.exe'
+    Invoke-WebRequest $url -OutFile $temp_file
+
+    $version = (Get-Command $temp_file).Version
```

Comment:
Interesting way to obtain the version. I like it.

However, its not appropriate here. The file size is ~27MB and the version is available on download page.

So you should find out something that produces faster update:
- Download page has info about the version.
- Blog seems to serve as release new information: http://www.pdfforge.org/blog, so you can take in their RSS feed (http://www.pdfforge.org/blog/rss) and filter items with PDFCreator in title and get the version.